includes/Modules/Analytics.php

```diff
@@ -325,6 +325,23 @@ protected function print_amp_gtag() {
 			),
 		);
 
+		/**
+		 * Filters configuration options for the object that is passed to amp-analytics as a JSON component.
+		 *
+		 * @since 1.0.0
+		 *
+		 * @see https://developers.google.com/gtagjs/devguide/amp
+		 *
+		 * @param array $gtag_amp_opt gtag amp options.
+		 */
+		$config = apply_filters( 'googlesitekit_amp_gtag_opt', $gtag_amp_opt );
+
+		// Ensure gtag_id is set to the correct value.
+		if ( ! is_array( $config ) ) {
+			return;
```

Comment:
@felixarntz should we set to original value if $config is not an array when it should be?
`$config = $gtag_amp_opt;` (using new naming $gtag_amp_opt_filtered) instead of empty array?

Reply:
Sounds good.